Web/ForgeTrust.Runnable.Web.RazorDocs/wwwroot/css/app.css

```
@@ -0,0 +1,321 @@
+
+.docs-content p {
+    margin: 0 0 0.75rem;
+}
+
+.docs-content pre {
+    background-color: #1e293b;
+    border-radius: 0.5rem;
+    padding: 0.9rem;
+    border: 1px solid #334155;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)